tests/components/plugwise/test_switch.py

```diff
@@ -17,6 +19,31 @@ async def test_adam_climate_switch_entities(hass, mock_smile_adam):
     assert str(state.state) == "on"
 
 
+async def test_adam_climate_switch_negative_testing(hass, mock_smile_adam):
+    """Test exceptions of climate related switch entities."""
+    mock_smile_adam.set_relay_state.side_effect = PlugwiseException
+    entry = await async_init_integration(hass, mock_smile_adam)
+    assert entry.state == ENTRY_STATE_LOADED
+
+    await hass.services.async_call(
+        "switch",
+        "turn_off",
+        {"entity_id": "switch.cv_pomp"},
+        blocking=True,
+    )
+    state = hass.states.get("switch.cv_pomp")
+    assert str(state.state) == "on"
+
+    await hass.services.async_call(
+        "switch",
+        "turn_on",
+        {"entity_id": "switch.fibaro_hc2"},
```

Comment:
is this entity id correct? seems off to me

Reply:
Maybe I should have chosen another 'plug' to test in hindsight, but this user apparently decided to switch their HomeCenter with another automation product 😄 Reason for picking these from the fixtures is that those are the same ones we test our module against (hence making sense we fully test the fixture on both module CI and the HA CI). A few lines up the previous test (the positive one) uses the same entities.

Reply:
[link/loc of fixture source](https://github.com/plugwise/python-plugwise/blob/6e340ce4957cc3415658ddf4ccb7c34046ad7090/userdata/adam_multiple_devices_per_zone/core.domain_objects.xml#L3255)

Reply:
thanks for explaining